b2share/modules/communities/api.py

```diff
@@ -91,12 +91,21 @@ def get(cls, id=None, name=None, with_deleted=False):
     @classmethod
     # TODO: change this into a search function, not just a list of communities
     # TODO: a query should be given
-    def get_all(cls, start, stop):
+    def get_all(cls, start=None, stop=None):
         """Searches for matching communities."""
-        start = int(start)
-        stop = int(stop)
         from .models import Community as CommunityMeta
-        metadata = CommunityMeta.query.order_by(CommunityMeta.created).limit(stop)[start:]
+        if (start==None and stop==None):
+            metadata = CommunityMeta.query.order_by(CommunityMeta.created)
+        elif (start and stop):
```

Comment:
`bool(start) == False` does not mean that `start is None`
[It is recommended.](http://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_none.html?highlight=none) to use `is None` or `is not None` in python.